app/invocation/invocation_execution_card.tsx

```diff
@@ -290,7 +290,7 @@ export default class ExecutionCardComponent extends React.Component {
                 <div>
                   <div
                     className="invocation-execution-row-digest"
-                    onClick={capabilities.action ? this.handleActionDigestClick.bind(this, execution) : undefined}>
+                    onClick={this.handleActionDigestClick.bind(this, execution)}>
```

Comment:
do we want to wait to enable this until we've got the localhost stuff figured out?